File: 程式語言 V2答案檔/PHP/407.php
<?php
    $file = fopen("read.txt", "r") or die("Unable to open file");
    $str = trim(fgets($file));
    $result = "";
    for($i = 0; $i < strlen($str); $i++){
        if($str[$i] == '*') continue;
        $result .= $str[$i];
    }
    $result_file = fopen("write.txt", "w") or die("Unable to create file");
    $result_file = fwrite($result_file, $result);
    fclose($result_file);
?>

File: Programming Language(ver.2)/C/407.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main()
{
    FILE * input;
    FILE * output;

    char * delim = "*";
    char * ptr_c;
    char str[1024];

    input = fopen("read.txt","rt");

    fgets(str,sizeof(str),input);
    ptr_c = strtok(str,delim);

    output = fopen("write.txt","wt");
    while(ptr_c != NULL){
        fprintf(output,"%s",ptr_c);
        ptr_c = strtok(NULL,delim);
    }
    
    fclose(input);
    fclose(output);
    return 0;
}
